dist/Exporter/Makefile.PL

```diff
@@ -0,0 +1,29 @@
+use 5.006;
+use ExtUtils::MakeMaker;
+
+my $EUMM_VERSION = $ExtUtils::MakeMaker::VERSION;
+$EUMM_VERSION =~ s/_//g;
+
+WriteMakefile(
+    NAME         => 'Exporter',
+    VERSION_FROM => 'lib/Exporter.pm',
+    ( $] > 5.011 ) ? () : ( INSTALLDIRS => 'perl' ),    # CPAN sourced versions should now install to site
+    PREREQ_PM => {
+        'Carp' => '1.05',
+    },
+    ( $] >= 5.005           ? ( ABSTRACT_FROM => 'lib/Exporter.pm' ) : () ),
```

Comment:
Obsolete, we require 5.006 above.

Reply:
Fixed, also standardized the INSTALLDIRS stanza.